src/client/utils/getTextFromChildren.util.tsx

```diff
@@ -0,0 +1,16 @@
+export function getTextFromChildren(node: unknown): string {
+	if (typeof node === 'string') {
+		return node;
+	}
+	if (typeof node === 'number') {
+		return node.toString();
+	}
+	if (node instanceof Array) {
+		return node.map(getTextFromChildren).filter(Boolean).join(' ');
+	}
+	if (typeof node === 'object' && node) {
+		const objNode = node as { props?: { children: unknown } };
```

Comment:
Si je mets pas cette ligne j'ai une erreur object does not have property "props" 🫤